Fix drag and drop issue in taurusform

mrosanes opened the pull request:
Use bytes instead of str for setting mime data.

cpascual commented:
I have some issues with this PR:

- it changes things 2 independent things in 2 different files. Two separated commits should be done.
- IIMHO, we should not fix this by handling non-quantities, but by detecting the case and giving a warning (i.e. wrapping the problematic line with a try-except and a warning message)
- If there is interest in actually handling non-quantities, then we should use try-except instead of `if isinstance` (to allow for duck-typing) and to avoid performance-penalty in the most common case of quantities being used.

So I am rejecting this PR.  Can you please address these issues and submit another one?

Commit:
Cast newstr to str

Cast newstr to string to be able to setData.
This fixes drag & drop in taurusforms.

Commit:
Use bytes for setting mime data

Use bytes instead of string for setting mime data.
